### PR TITLE
Refractoring/contracts

### PR DESCRIPTION
**Pull request by helderjnpinto**
## Description

Just changed unecessary else return blocks, fix typos in comments, and added a default in switch for throwing a not implemented method error. 

## Type of change

Refractoring code

## Checklist:

- [x] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran ```npm run dtslint``` with success and extended the tests and types if necessary.
- [x] I ran ```npm run test:unit``` with success and extended the tests if necessary.
- [x] I ran ```npm run build-all``` and tested the resulting file/'s from  ```dist``` folder in a browser.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the live network.

### CODE REVIEW

**Comment by nivida** [comment]
@helderjnpinto Thanks for applying those required improvements! I will review it tomorrow 💪

File: packages/web3-eth-contract/src/index.js [hunk 1]
```diff
@@ -877,131 +875,134 @@ Contract.prototype._executeMethod = function _executeMethod(){
...
+                defaultHardfork: _this._parent.defaultHardfork,
+                handleRevert: _this._parent.handleRevert,
+                extraFormatters: extraFormatters,
+                abiCoder: abi
+            })).createFunction();
+
+            return sendTransaction(args.options, args.callback);
+
+        default:
+            throw new Error('Method "' + args.type + '" not implemented.');
```
Comment (nivida) [inline packages/web3-eth-contract/src/index.js]: Could you if you open another PR for this project reference additions like this newly added error in the description of the pull request?

File: packages/web3-eth-contract/src/index.js [hunk 2]
```diff
@@ -877,131 +875,134 @@ Contract.prototype._executeMethod = function _executeMethod(){
...
 
     }
 
+
```
Comment (nivida) [inline packages/web3-eth-contract/src/index.js:1004]: Those line breaks aren't required and can get removed.
```suggestion
```

File: packages/web3-eth-contract/src/index.js [hunk 3]
```diff
@@ -877,131 +875,134 @@ Contract.prototype._executeMethod = function _executeMethod(){
...
+
```
Comment (nivida) [inline packages/web3-eth-contract/src/index.js]:
```suggestion
```

**Review by nivida — CHANGES_REQUESTED** [review]
LGTM! I've just added a suggestion to remove unnecessary line breaks.

**Review by cgewecke — APPROVED** [review]
LGTM! Thanks @helderjnpinto.

**Review by nivida — APPROVED** [review]

**Pull request merged**